specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/examples/NetworkManagerGroupPut.json

```diff
@@ -0,0 +1,71 @@
+{
+  "title": "Create/Update Azure Virtual Network Manager Network Group",
+  "parameters": {
+    "api-version": "2020-08-01",
+    "subscriptionId": "subscriptionC",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager",
+    "networkGroupName": "TestNetworkGroup",
+    "parameters": {
+      "properties": {
+        "displayName": "My Network Group",
+        "description": "A sample group",
+        "memberType": "VirtualNetwork",
+        "groupMembers": [
+          {
+            "resourceId": "/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/virtualnetworks/vnet1"
+          }
+        ],
+        "conditionalMembership": ""
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "ETag": "be3cfaa2-3d02-4624-92d7-a26766306c8b"
+      },
+      "body": {
+        "name": "TestNetworkGroup",
+        "id": "/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/networkGroups/TestNetworkGroup",
+        "type": "Microsoft.Network/networkManager/networkGroups",
```

Comment:
nit: Microsoft.Network/networkManagers/networkGroups
Please check type usage in other places too.